File: ASOS-Frontend/DTOs/ManageAccount/EmailChangeDto.cs
using System.Text.Json.Serialization;

namespace DTOs.ManageAccount
{
	public class EmailChangeDto
	{
		[JsonPropertyName("email")]
		public string Email { get; set; }

		[JsonPropertyName("newEmail")]
		public string NewEmail { get; set; }

		[JsonPropertyName("password")]
		public string Password { get; set; }
	}
}

File: src/main/java/com/asos/reservationSystem/domain/dto/EmailChangeDto.java
package com.asos.reservationSystem.domain.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EmailChangeDto {
    private String email;
    private String newEmail;
    private String password;
}
